funcx_endpoint/tests/version_mismatch_tests/config.py

```diff
@@ -0,0 +1,32 @@
+from funcx_endpoint.endpoint.utils.config import Config
+from funcx_endpoint.executors import HighThroughputExecutor
+from parsl.providers import LocalProvider
+import os
+CONDA_ENV = os.environ["WORKER_CONDA_ENV"]
+print(f"Using conda env:{CONDA_ENV} for worker_init")
+
+config = Config(
+    executors=[HighThroughputExecutor(
+        provider=LocalProvider(
+            init_blocks=1,
+            min_blocks=0,
+            max_blocks=1,
+            worker_init=("source ~/anaconda3/etc/profile.d/conda.sh; "
```

Comment:
this is probably something specific to your development environment - I don't have an ~/anaconda3